output/json.go

```diff
@@ -0,0 +1,74 @@
+package output
+
+import (
+	"encoding/json"
+	"github.com/yarpc/crossdock/execute"
+	"github.com/yarpc/crossdock/plan"
+	"io/ioutil"
+)
+
+var StatusNames = map[execute.Status]string{
+	execute.Success: "success",
+	execute.Failed:  "failed",
+	execute.Skipped: "skipped",
+}
+
+type JsonTestReport struct {
```

Comment:
It's an angry identifier, `XMLHTTPRequest`. However, since there's packages, you would use that to namespace your identifiers into something like: `xml/http` and have a `http.Request`, or `xmlhttp.Request`